plugins/woocommerce/src/Gateways/PayPal/WebhookHandler.php

```diff
@@ -1,44 +1,43 @@
 <?php
 /**
- * Class WC_Gateway_Paypal_Webhook_Handler file.
+ * Class WebhookHandler file.
  *
  * @package WooCommerce\Gateways
  */
 
 declare(strict_types=1);
 
+namespace Automattic\WooCommerce\Gateways\PayPal;
+
+use Automattic\WooCommerce\Gateways\PayPal\Constants as PayPalConstants;
+use Automattic\WooCommerce\Gateways\PayPal\Helper as PayPalHelper;
 use Automattic\WooCommerce\Enums\OrderStatus;
 
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
-if ( ! class_exists( 'WC_Gateway_Paypal_Helper' ) ) {
-	require_once __DIR__ . '/class-wc-gateway-paypal-helper.php';
-}
-
 if ( ! class_exists( 'WC_Gateway_Paypal_Request' ) ) {
 	require_once __DIR__ . '/class-wc-gateway-paypal-request.php';
```

Comment:
```suggestion
	require_once WC_ABSPATH . 'includes/gateways/paypal/includes/class-wc-gateway-paypal-request.php';
```

Until the request class is moved.